docs/v12/documentation_routes.js

```diff
@@ -19,7 +19,12 @@ router.all('*', function (req, res, next) {
 
 // Docs index
 router.get('/', function (req, res) {
-  res.render('index')
+  if (!req.originalUrl.endsWith('/')) {
```

Comment:
This has a bug in it.  `http://localhost:3000/docs?abc=def` redirects to `http://localhost:3000/docs?abc=def/`.  I've tried this solution and it works for me:

```
router.get('/', function (req, res) {
  const [url, originalQueryString] = req.originalUrl.split('?')
  if (!url.endsWith('/')) {
    const queryString = originalQueryString ? `?${originalQueryString}` : ''
    // redirect to /docs/ so we can use relative hrefs safely
    res.redirect(`${url}/${queryString}`)
  } else {
    res.render('index')
  }
})
```

Reply:
Good catch